test/unit/sqs/index.test.ts

```diff
@@ -244,16 +254,56 @@ describe("SqsQueue tests", () => {
 
 			mockErrorHandler.mockImplementation((error: Error, context: Context<TestMessage>) => {
 
-				expect(context.payload.msg).toBe(testPayload.msg);
-				expect(error).toBeInstanceOf(SqsTimeoutError);
+				try {
+					expect(context.payload.msg).toBe(testPayload.msg);
+					expect(error).toBeInstanceOf(SqsTimeoutError);
```

Comment:
Turns out if expect fails here it doesn't fail the test (cuz exception doesn't reach the top level). So we have to handle it and call fail explicitly